sprint1/koko-earing-bananas/KokoEating.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+
+public class KokoEating {
+
+    public static void main(String[] args) {
+        System.out.println(minEatingSpeed(new int[]{3,6,7,11}, 8));
+    }
+
+    public static int minEatingSpeed(int[] piles, int h) {
+        Arrays.sort(piles);
+        return binarySearch(piles, h);
+    }
+
+    public static int hoursToEat(int[] piles, int speed) {
+
+        int hours = 0;
+        for (int pile : piles) {
+            hours += (int) Math.ceil((double) pile / speed);
+        }
+        return hours;
+
+    }
+
+    public static int binarySearch(int[] piles, int hours) {
```

Comment:
Consider to gave the proper naming for the function. That's not actually a binary search :)